app-ui/src/bundles/Auth/selectors/AuthSelectors.js

```diff
@@ -0,0 +1,24 @@
+
+//  Map Request state to Button props;
+export const requestButtonProps = formInvalid => r => ({
+  disabled: formInvalid || r.data.pending,
+  loading: r.data.pending,
+});
+
+// Get activate account data from state
+export const getActivateAccountEmail = state => state.auth.activateAccount;
+
+//  Get SignIn form data from state
+export const getSignInForm = state => state.auth.signIn.form;
+
+//  Get SignUp form from state
+export const getSignUpForm = state => state.auth.signUp.form;
+
+//  Get validation token from url in props
+export const getResetToken = ownProps => ownProps.match.params.token;
+
+//  Get ResetPassword form from state
+export const getResetPasswordForm = state => state.auth.resetPassword.form;
+
+//  Get RecoverPassword form from state
+export const getRecoverPasswordForm = state => state.auth.recoverPassword.form;
```

Comment:
:+1:  for using selectors

Reply:
> @orar Code looks good so far +1 I, need some days to check the functionality.
> 
> Is the file node_modules/questrar really needed?

File?

Reply:
Its a generated link file in the first commit `back to work`. Not needed. Its omitted.